db/migrate/20180117194839_add_place3_optional_flag.rb

```diff
@@ -0,0 +1,5 @@
+class AddPlace3OptionalFlag < ActiveRecord::Migration[5.1]
+  def change
+    add_column :events, :has_place_3_match, :boolean, default: true
```

Comment:
Are you sure this is an acceptable name?

Reply:
it is acceptable. Maybe there are better names. Proposals?

Reply:
`third_place_play-off` maybe? Other suggestions could be the equivalent of "kleines Finale" which I don't know if it exists in english :P

Reply:
without the "has" it sounds like we are referencing a match object, not a boolean. And the difference between, place_3_match, third_place_match or third_place_play_off isn't too big, imo